loop_closure:
  max_icp_iterations: 100
  max_dist_lc: 10.0     # in m
  min_traveled_lc: 4.0 # in m
  dist_between_lcs: 0.0 # in m

  max_prereg_icp_fitness: 0.1 # in m²
  max_lc_icp_fitness: 0.3 # in m²

  do_preregistration: false

  prior_rotation_noise_x: 0.01
  prior_rotation_noise_y: 0.01
  prior_rotation_noise_z: 0.01 #3.14159
  prior_translation_noise_x: 0.064
  prior_translation_noise_y: 0.064
  prior_translation_noise_z: 0.064

  between_rotation_noise_x: 0.01
  between_rotation_noise_y: 0.01
  between_rotation_noise_z: 0.01
  between_translation_noise_x: 0.064
  between_translation_noise_y: 0.064
  between_translation_noise_z: 0.064

map:
  # directory to save map in
  dir: "/home/patrick/maps/generated"

  comment: "slam6d_listening"

  # Truncation distance for the TSDF values (in m)
  max_distance: 0.6

  # Size of one grid cell in the discretised map (in m per cell)
  resolution: 128 #0.1 #0.03

  # localmap size for every dimension (in m)
  size:
    x: 40
    y: 40
    z: 25

  # Maximum weight for one TSDF cell. This variable ensures that later changes can still have an influence to the map
  max_weight: 10.0

ray_tracer:
  #general ray_tracer stuff
  hor_res: 1024
  vert_res: 128
  opening_degree: 45
  step_size: 0.032
  ray_size: 0.01

  # icp related stuff
  hor_res_icp: 256
  vert_res_icp: 64
  opening_degree_icp: 90
